Dockerfile

```diff
@@ -9,6 +9,7 @@ FROM debian:stretch-20190812-slim
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
         iptables \
+        procps \
```

Comment:
Just curious: can we not set the value without this?

Reply:
this gives us the `sysctl` command